### ISSUE TITLE
Add command to makefile for turning preprint latex into PDF

### ISSUE THREAD

**Issue by cthoyt** [openjournals/inara#76]
It's a bit inconvenient that the Makefile's command `preprint` creates a tex file but does not in turn compile it to PDF. it would be nice to have this as an option

**Comment by AaronFeller** [comment]
Agreed! I was trying to sort this out just now. I thought I had an unlogged error.
